plugins/module_utils/vmware_rest_client.py

```diff
@@ -21,25 +21,15 @@
     REQUESTS_IMP_ERR = traceback.format_exc()
     HAS_REQUESTS = False
 
-PYVMOMI_IMP_ERR = None
-try:
```

Comment:
It may, but we agreed to start with only relevant needed functionality, same for the other module_utils files. We would need to fix it there as well, but on forum we agreed on this approach.